data/json/items/book/fabrication.json

```diff
@@ -136,7 +136,7 @@
     "skill": "fabrication",
     "required_level": 3,
     "max_level": 5,
-    "proficiencies": [ { "proficiency": "prof_bowyery", "time_factor": 0.75, "fail_factor": 0.5 } ],
+    "proficiencies": [ { "proficiency": "prof_bowyery", "time_factor": 0.15, "fail_factor": 0.5 } ],
```

Comment:
Either this one shouldn't be quite this good (I'd go with fail_factor 0.45), or the Fletcher's Friend should be a bit better (I'd go with fail_factor 0.4), considering that they're only 1 level apart.